src/dlio/odom.cc

```diff
@@ -175,15 +174,25 @@ void dlio::OdomNode::getParams() {
   ros::param::param<std::string>("~dlio/frames/lidar", this->lidar_frame, "lidar");
   ros::param::param<std::string>("~dlio/frames/imu", this->imu_frame, "imu");
 
+
   // Get Node NS and Remove Leading Character
   std::string ns = ros::this_node::getNamespace();
   ns.erase(0,1);
 
-  // Concatenate Frame Name Strings
-  this->odom_frame = ns + "/" + this->odom_frame;
-  this->baselink_frame = ns + "/" + this->baselink_frame;
-  this->lidar_frame = ns + "/" + this->lidar_frame;
-  this->imu_frame = ns + "/" + this->imu_frame;
+  // Concatenate Frame Name Strings (if there is one)
+  if (ns.compare("") != 0 && !ns.empty()) {
+    //Append to frames if we have namespace
+    this->odom_frame = ns + "/" + this->odom_frame;
+    this->baselink_frame = ns + "/" + this->baselink_frame;
+    this->lidar_frame = ns + "/" + this->lidar_frame;
+    this->imu_frame = ns + "/" + this->imu_frame;
+  }
+  std::cout << "odom_frame: " << this->odom_frame << std::endl;
```

Comment:
Why are you printing stuff, if it was for debug remove before push to master

Reply:
This will only be printed once, so not a big deal